src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,99 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+import { MovieCard } from '../MovieCard';
+import { getMovie } from '../../api';
+
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movies: Movie[];
+  setMovies: (movies: Movie[]) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ movies, setMovies }) => {
+  const [query, setQuery] = useState<string>('');
```

Comment:
you actually don't need to use generics for literals, the types are generated automatically based on the default value
```suggestion
  const [query, setQuery] = useState('');
```